extensions/wikia/FeedsAndPosts/FeedsAndPostsController.class.php

```diff
@@ -16,9 +17,15 @@ public function getTopArticles() {
 
 	public function getAll() {
 		$this->response->setValues( [
-			'recentChanges' => ( new RecentChanges() )->get(),
-			'topArticles' => ( new TopArticles() )->get(),
+				'recentChanges' => ( new RecentChanges() )->get(),
```

Comment:
Is this indentation OK? Seems like it has too many tabs (or maybe my github view renders it badly).